### PR TITLE
Implement pangram exercise

### PR DESCRIPTION
**Pull request by benreyn**

### CODE REVIEW

**Comment by yurrriq** [comment]
LGTM but maybe needs a rebase

**Comment by benreyn** [comment]
Rebased on master, taking a look at failing CI.

**Comment by benreyn** [comment]
@yurrriq, It looks like the version of emacs we use on TravisCI to run our examples is < 25 and it appears that [`seq-every-p`](https://github.com/emacs-mirror/emacs/blob/master/lisp/emacs-lisp/seq.el#L311) was not included versions that old. Personally, I would recommend we move to a newer version on CI, but if you would like, I can either...
A. rewrite my example for `pangram` to not use `seq-every-p`
or
B. re-implement the function in my example.

Alternatively, if you would like up the version of emacs we use on CI, I could submit that as a separate PR doing so (which seems best to me in this case) or simply add a commit to this branch that does so.

**Comment by yurrriq** [comment]
:+1: to upgrading Emacs. Your call whether that land win this branch or its own.

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 language: bash
 before_install:
-  - sudo add-apt-repository -y ppa:cassou/emacs # Emacs 24.3
+  - sudo add-apt-repository -y ppa:kelleyk/emacs # Emacs 25
   - sudo apt-get update
-  - sudo apt-get install -y emacs24-nox
+  - sudo apt-get install -y emacs25
```
Comment (yurrriq) [inline .travis.yml]: Is there an `emacs25-nox` package?

**Comment by benreyn** [comment]
@yurrriq, It took a bit of tweaking, but I got this passing on CI. Im not quite sure what was going wrong with `(require 'seq)`. It was loading the library but also throwing an error that would cause the test command to exit non-zero and therefore fail the tests. (Take a look [here](https://travis-ci.org/exercism/elisp/builds/326885327#L730)) I rewrote using `cl-lib` and everything seems to be working fine, and I squashed my playing around on CI down to a reasonable number of commits. 

Flagging this for the future though, that maybe the CI setup deserves a more intentional pass to make implementing examples a little easier on contributors. Maybe we shouldn't need to manually require emacs built-in libraries. Im thinking some sort of bare-bones `init.el`. Let me know what you think, and I can log an issue and start working on it if you like the idea.

**Comment by yurrriq** [comment]
I don't have much experience running elisp on Travis, so I'm happy to review whatever you come up with.
